pyccel/codegen/printing/ccode.py

```diff
@@ -798,9 +811,9 @@ def get_print_format_and_arg(self, var):
                           ('complex',8) : '(%.12lf + %.12lfj)',
                           ('complex',4) : '(%.12f + %.12fj)',
                           ('int',4)     : '%d',
-                          ('int',8)     : '%ld',
-                          ('int',2)     : '%hd',
-                          ('int',1)     : '%c',
+                          ('int',8)     : LiteralString("%") + CMacro('PRId64'),
+                          ('int',2)     : LiteralString("%") + CMacro('PRId16'),
+                          ('int',1)     : LiteralString("%") + CMacro('PRId8'),
                           ('bool',4)    : '%s',
                           ('string', 0) : '%s'}
```

Comment:
Would this dictionary be better as a global object rather than recreating it at every call to `get_print_format_and_arg`?

Reply:
yes we could do that, that would match the [Fortarn printer](https://github.com/pyccel/pyccel/blob/b1cd3459d87c18a5267c31d0833654724b6a92ce/pyccel/codegen/printing/fcode.py#L176-L183) too

Reply:
It should be faster